nextstrain/cli/runner/managed_conda.py

```diff
@@ -0,0 +1,337 @@
+"""
+Run commands with access to a fully-managed Conda environment.
+"""
+
+import os
+import platform
+import requests
+import shutil
+import subprocess
+import tarfile
+import traceback
+from pathlib import Path, PurePosixPath
+from typing import Iterable, Tuple
+from urllib.parse import urljoin
+from ..types import RunnerSetupStatus, RunnerTestResults, RunnerUpdateStatus
+from ..util import capture_output, exec_or_return, user_app_data_dir, warn
+
+
+USER_APP_DATA_DIR = user_app_data_dir()
+
+PREFIX     = USER_APP_DATA_DIR / "runtimes/managed-conda"
```

Comment:
What is this `string / string` syntax in python called? I've not seen it before

Reply:
Ah, `USER_APP_DATA_DIR` here is a `Path` object from the [pathlib module](https://docs.python.org/3/library/pathlib.html). It overloads the `/` operator (by defining a `__truediv__(self, other)` method) to perform path joining/concatenation and accepts a string or another `Path` object as the RHS.